CHANGELOG

```diff
@@ -25,6 +25,8 @@ qtile X.X.X, released XXXX-XX-XX:
           AND-semantics, allowing for more complex and specific rules
         - Python 3.9 support
         - switch to Github Actions for CI
+        - added `only_focused` setting to Max-layout, allowing to draw multiple
+          clients on top of each other when set to False
```

Comment:
Can you add a "Fixes #415" tag to the commit message here?

Reply:
You mean the commit message or also the CHANGELOG?

I also added a "Closes #753" to the first commit message.